js/apps/viewer/src/app/animation-player/animation-player.tsx

```diff
@@ -208,6 +211,12 @@ export class AnimationPlayer extends PureComponent<Props> {
                             <Typography variant="caption">
                                 {currentMs} / {totalMs} {animationParams.unit}
                             </Typography>
+                            <span className={classes.spacer}></span>
+                            <AnimationSpeedController
+                                    delta={delta}
```

Comment:
Use 4 spaces for indentation for all these comp. props 😉 
```suggestion
                                delta={delta}
```